src/components/InventoryTable/InventoryList.scss

```diff
@@ -18,6 +18,10 @@
     .ins-c-chip-filters .pf-v5-c-chip-group:not(:last-of-type) { margin-right: var(--pf-v5-global--spacer--xs); }
 
     .ins-c-tagfilter { width: initial; }
+
+    .ins-c-tagfilter.pf-v5-c-menu-toggle {
+        height: 36px;
```

Comment:
I think it is better to use some PF variables instead of 36px. Because, this may work for certain screen resolution, but not for others.

Reply:
Also, the issue should be related to something not properly handled with regards to the tag filter dropdown. As this issue happens to tag filter only, not others.

Reply:
@mkholjuraev thanks for the review! I agree with you. Change has to be done on FEC side so closing this PR